ktext.md

```diff
@@ -312,12 +312,11 @@ However ProperTree will handle this for you, so you need not concern yourself
 ### Extras
 
 * [AppleMCEReporterDisabler](https://github.com/acidanthera/bugtracker/files/3703498/AppleMCEReporterDisabler.kext.zip)
-  * Useful starting with Catalina to disable the AppleMCEReporter kext which will cause kernel panics on AMD CPUs and dual-socket systems
-  * Affected SMBIOS:
+  * Required in macOS 12.3 and above on AMD systems and dual-socket systems. Do not use if on macOS 12.2.1 or below.
+  * Affected SMBIOSes:
```

Comment:
Do you know if it's needed in Catalina or newer for dual socket systems?

Reply:
It was only needed for a couple 10.15.0 betas. It was fixed in the 10.15.0 release iirc.

Reply:
The information should be accurate now.

Reply:
> Do not use if on macOS 12.2.1 or below.

Are you sure about this?

Reply:
It doesn't say that anymore @dhinakg , check the Files Changed tab

Reply:
Ah I see

Reply:
It wouldn't be wrong if it did say that though. AMD hasn't needed AppleMCEReporterDisabler (afaik) since a certain 10.15.0 beta. One of us could ask Pavo if you want confirmation of that, he'd know for sure.

Reply:
Found this: @dhinakg 
https://discord.com/channels/249992304503291905/263757191608139779/914934705126768680